src/pedalmarking.ts

```diff
@@ -33,12 +33,19 @@ function drawPedalGlyph(name: string, context: RenderContext, x: number, y: numb
  */
 export class PedalMarking extends Element {
   /** To enable logging for this class. Set `Vex.Flow.PedalMarking.DEBUG` to `true`. */
-  static DEBUG: boolean;
+  static DEBUG: boolean = false;
 
   static get CATEGORY(): string {
     return 'PedalMarking';
   }
 
+  static TEXT_FONT: Required<FontInfo> = {
```

Comment:
The main Element class gets a default TEXT_FONT which specifies the font family, size, weight, and style. Each subclass may optionally override this default.